src/forklift/cli.py

```diff
@@ -29,13 +29,14 @@
 
 log = logging.getLogger('forklift')
 template = join(abspath(dirname(__file__)), 'report_template.html')
+config_location = join(abspath(dirname(__file__)), '..', 'config.json')
```

Comment:
👍 
I'm guessing this will prevent issues when arcpy changes the working directory.